### PR TITLE
passed tests, ran machine, blew up with unexpected behavior

### PR DESCRIPTION
**Pull request by codyargyle**
PR for Misael

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# lab2
-lab2 imbedded 2
-
+Cody Argyle & Misael Nava
+Lab2
```
Comment (TechnicallyRy) [inline README.md:2]: maybe add a short description of the lab? not required but could be useful

File: lib/static_linking/include/uart_static.h [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#pragma once
+#ifdef TESTING_ENV
+struct device{};
+#else
+#include <drivers/uart.h>
+#endif
```
Comment (TechnicallyRy) [inline lib/static_linking/include/uart_static.h:6]: good environment abstractions

**Review by TechnicallyRy — APPROVED** [review]
Nice job, good to merge

**Comment by Dima5877** [comment]
Yes, this should work.

**Review by Fredware — APPROVED** [review]
I would remove editor files from version control (e.g., .vscode), but other than that, it looks okay.

**Pull request merged**